Source/CsDebugScript.Engine/Engine/Utility/SimpleCache.cs

```diff
@@ -70,6 +70,9 @@ public T Value
         {
             get
             {
+                // value = populateAction();
+                // return value;
+
                 if (!Cached)
```

Comment:
Removed.

---
In reply to: [69628266](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69628266) [](ancestors = 69628266)